common/presentation/src/main/kotlin/com/pandulapeter/myTodos/presentation/MyTodosApp.kt

```diff
@@ -7,12 +7,14 @@ import com.pandulapeter.myTodos.presentation.resources.Color
 import com.pandulapeter.myTodos.presentation.ui.DemoPicker
 
 @Composable
-fun MyTodosApp() = MaterialTheme(
-    colors = lightColors(
-        primary = Color.brand,
-        primaryVariant = Color.brand,
-        secondary = Color.brand,
-        secondaryVariant = Color.brand
-    ),
-    content = { DemoPicker() }
-)
\ No newline at end of file
+fun MyTodosApp() {
+    MaterialTheme(
```

Comment:
This applies in general to usages of expression body: 
In my opinion expression body should be used only when the intention is to return a value. While you `Unit` is a valid type to be returned in this case the method returns Unit because it doesn't return something.